contracts/protocol/pool/Pool.sol

```diff
@@ -486,6 +487,63 @@ contract Pool is ReentrancyGuard, VersionedInitializable, PoolStorage, IPool {
         );
     }
 
+    function auctionLiquidationStart(
+        address token,
+        uint256 tokenId,
+        address user
+    ) external virtual {
+        AuctionLiquidationLogic.executeAuctionLiquidationStart(
+            _auctions,
+            _reserves,
+            _reservesList,
+            _usersConfig,
+            DataTypes.ExecuteAuctionLiquidationStartParams({
+                reservesCount: _reservesCount,
+                collateralTokenId: tokenId,
+                collateralAsset: token,
+                user: user,
+                priceOracle: ADDRESSES_PROVIDER.getPriceOracle()
+            })
+        );
+    }
+
+    function auctionLiquidationBid(
+        address token,
+        uint256 tokenId,
+        uint256 amount
+    ) external virtual {
+        AuctionLiquidationLogic.executeAuctionLiquidationBid(
```

Comment:
@WalidOfNow should we move the ETH into the pool here?

Reply:
the eth should has already been sent to the pool and can be checked in the `msg.value`